Code/Framework/AzFramework/AzFramework/DocumentPropertyEditor/DocumentSchema.h

```diff
@@ -276,6 +276,18 @@ namespace AZ::DocumentPropertyEditor
                 return {};
             }
 
+            // This case is for handling legacy EnumValueKey attributes
+            if constexpr (AZStd::is_same_v<GenericValueType, uint64_t>)
```

Comment:
It is better to use an `AZ::u64` as that is always an `unsigned long long` across all of O3DE support platforms, where as `uint64_t` has different underlying types on Linux and Android vs Windows and Mac(`unsigned long` vs `unsigned long long`)

This matters because the Uuid is different for those types: https://github.com/o3de/o3de/blob/development/Code/Framework/AzCore/AzCore/RTTI/TypeInfo.h#L1305-L1306

Reply:
Understood. I thought it was the other way around. Will get that changed here and throughout.